samples/suit/smp_transfer/VERSION

```diff
@@ -0,0 +1,20 @@
+VERSION_MAJOR = 0
+VERSION_MINOR = 1
+PATCHLEVEL = 0
+VERSION_TWEAK = 0
+
+
+app_root_seq_num = 1
```

Comment:
not sure I like half the file being in caps lock and half being without, also can see here that strings should not need quotes: https://github.com/zephyrproject-rtos/zephyr/commit/fd0e46506fabd4c0c91ecb336a39cb6a6aa2b270

Reply:
Reworked to:
 - Use uppercase names
 - Remove quotes